sdcm/tester.py

```diff
@@ -326,6 +326,19 @@ def run(self, result=None):
         result = super(ClusterTester, self).run(self.result)
         return result
 
+    @property
+    def reliable_replication_factor(self) -> int:
+        """
+        Calculates reliable replication factor for the test to run a load on
+        :return:
+        """
+        n_db_nodes = self.params.get('n_db_nodes')
```

Comment:
What do you think about converting it to string and removing `if`'s?

Reply:
done